tests/test_snapshot_download.py

```diff
@@ -88,3 +90,60 @@ def test_download_model(self):
 
             # folder name contains the revision's commit sha.
             self.assertTrue(self.first_commit_hash in storage_folder)
+
+    def test_download_private_model(self):
+        self._api.update_repo_visibility(self._token, REPO_NAME, private=True)
```

Comment:
You might want to update it back to public no?

Reply:
Good catch!

Reply:
Done!